components/collapsible/composed/src/index.scss

```diff
@@ -0,0 +1,38 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+.sui-CollapsibleComposed {
+  background-color: $c-white;
+
+  &-label {
+    border-bottom: dotted 1px $c-black;
+    font-size: $fz-l;
+    font-weight: $fw-bold;
+    width: 100%;
+  }
+
+  &-content {
+    padding-left: $p-m;
+  }
+
+  &-item {
+    display: block;
+    font-size: $fz-s;
+
+    &--inline {
+      display: inline;
+      margin-right: $m-s;
+    }
+  }
+
+  &-toggler {
+    background: $c-primary-light;
+    border-radius: 4px;
+    border: solid 1px $c-primary;
+    box-shadow: inset 0 1px 0 rgba(255, 255, 255, 0.4), 0 1px 1px rgba(0, 0, 0, 0.2);
```

Comment:
Your linter should warn you about not using leading zeroes. Use ' .4' instead

Reply:
linter did not warn, but you're right.